spec/update-process-env-spec.js

```diff
@@ -124,30 +182,42 @@ describe('updateProcessEnv(launchEnv)', function () {
     })
 
     describe('shouldGetEnvFromShell()', function () {
-      it('returns the shell when the shell should be patched', function () {
+      it('indicates when the environment should be fetched from the shell', function () {
         process.platform = 'darwin'
-        expect(shouldGetEnvFromShell('/bin/sh')).toBe(true)
-        expect(shouldGetEnvFromShell('/usr/local/bin/sh')).toBe(true)
-        expect(shouldGetEnvFromShell('/bin/bash')).toBe(true)
-        expect(shouldGetEnvFromShell('/usr/local/bin/bash')).toBe(true)
-        expect(shouldGetEnvFromShell('/bin/zsh')).toBe(true)
-        expect(shouldGetEnvFromShell('/usr/local/bin/zsh')).toBe(true)
-        expect(shouldGetEnvFromShell('/bin/fish')).toBe(true)
-        expect(shouldGetEnvFromShell('/usr/local/bin/fish')).toBe(true)
+        expect(shouldGetEnvFromShell({SHELL: '/bin/sh'})).toBe(true)
```

Comment:
These were removed in a prior change and that resulted in a regression for zsh. While these may be perceived as redundant, I think they test for the vast majority of shells used by users, and so we should leave the assertions in there to guarantee no regressions occur.